docs/GKE.md

```diff
@@ -0,0 +1,72 @@
+# GKE
+
+This repository provides support for running the backend applications on GKE.
+
+## Setup
+
+### Terraform
+The provided Terraform [gke.tf](../infrastructure/gke.tf) will provision a [GKE Autopilot](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-overview) cluster. This method of provisioning GKE will automatically manage the nodes for the cluster as the backend applications are added.
+
+These terraform [services.tf](../infrastructure/services.tf) file enables the required cloud API services, such as `cloudbuild.googleapis.com` and `container.googleapis.com`.
+
+Additionally, there is a Cloud Spanner service account created for the backend services.
+
+### Kubectl
+To interact with the GKE cluster, ensure kubectl is installed.
+
+Once that is done, authenticate to GKE with the following commands:
+
+```
+export USE_GKE_GCLOUD_AUTH_PLUGIN=True
+export GKE_CLUSTER=cymbal-games-gke # change this based on the terraform configuration
+gcloud container clusters get-credentials $GKE_CLUSTER --region us-central1
+kubectl get namespaces
```

Comment:
Any particular reason for fetching namespaces? Would getting nodes / services / pods / etc be just as good? I personally like getting nodes because you can tell from the node names that you are connecting to the correct cluster.

Reply:
Namespaces is what is used in https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl so I just followed that.